restapi/configure_deathstar.go

```diff
@@ -84,7 +96,40 @@ func configureAPI(api *operations.DeathstarAPI) http.Handler {
 		return operations.NewPutExhaustPortServiceUnavailable().WithPayload(backtrace)
 	})
 	api.PostRequestLandingHandler = operations.PostRequestLandingHandlerFunc(func(params operations.PostRequestLandingParams) middleware.Responder {
-		return operations.NewPostRequestLandingOK().WithPayload("Ship landed\n")
+		var landingRequestURL, ok = os.LookupEnv("LANDING_REQUEST_URL")
+		if !ok {
+			return operations.NewPostRequestLandingOK().WithPayload(landingResponse() + "\n")
+		} else {
+			jsonBody := []byte(``)
+			bodyReader := bytes.NewReader(jsonBody)
+
+			nr, _ := http.NewRequest(http.MethodPost, landingRequestURL, bodyReader)
+			nr.Header = params.HTTPRequest.Header.Clone()
+			client := http.Client{
+				Timeout: 30 * time.Second,
+			}
+
+			res, err := client.Do(nr)
```

Comment:
Is the intent here to just allow the response to be customized, or to change the deathstar to do a whole other query when this endpoint is queried? I see in the description the bit about a redirect, but I would have thought that a redirect is just a different response code `302` with the `landingRequestURL` in the response.

Reply:
Sorry the redirect is a misexplanation. I'll update the description.
 
the LANDING_REQUEST_URL support as implemented lets me construct a simplified microservice-like interaction demo where the landing request functionality has been centralized as an internal cluster service not exposed outside the cluster.  

In my conference demos it let me thematically build towards gatewayAPI East-West capability discussion east-west service communication.